src/include/dng/hts/vcf.h

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2015 Reed A. Cartwright
+ * Authors:  kdai1 <kdai1@asu.edu>
+ *
+ * This file is part of DeNovoGear.
+ *
+ * DeNovoGear is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation; either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+#ifndef CXX_HTS_VCF_H
+#define CXX_HTS_VCF_H
+
+#include "dng/hts/hts.h"
+#include "dng/matrix.h"
+
+#include <vector>
+#include <string>
+#include <set>
+#include <htslib/vcf.h>
+
+#define VCF_VERSION "VCFv4.1"
+#define SOURCE "Denovogear 1.0.1"
+
+namespace hts { namespace vcf { 
```

Comment:
This should probably be in a "bcf" namespace since it wraps the bcf objects in htslib.